### ISSUE TITLE
MemoryError

### ISSUE THREAD

**Issue by z3r0burn** [Flexget/Flexget#1647]
### Expected behaviour:
clean run through config, download torrents and podcasts

### Actual behaviour:
takes upto 20 minutes to download a podcast using the download plugin or randomly crashes with MemoryError. Issue started after upgrading to 2.9.3. Tried upgrading to 2.9.4 and 2.9.5 with same results. I think i was running a 6 moth old version before upgrade but don't remember for sure the exact version number. after upgrading to 2.9.5 noticed same error would sometimes pop up in my TV shows task which passes the magnet/torrent to Deluge for downloading and doesn't actually download the files itself.

### Steps to reproduce:
- Step 1: ...
i just have to run my config

#### Config:
```
config.yml
schedules:
  - tasks: 'Podcasts'
    schedule:
      hour: 1

tasks:
  Podcasts:
    seen: local
    include: Podcasts.yml
    notify:
      task:
        title: Podcasts
        template: dark.template
        via:
          - email:
              from: XXX@gmail.com
              to: XXX@gmail.com
              smtp_host: smtp.gmail.com
              smtp_port: 587
              smtp_username: XXX
              smtp_password: XXX
              smtp_tls: yes
    priority: 5

podcasts.yml
pathscrub: windows
template: no_global
accept_all: yes

series:
  - Up and At Em with Jack and Ben:
      identified_by: sequence
      set:
        path: 'C:/Media/Pool/Downloads/Downloaded/temp/{{series_name}}/{{series_name}} - {{series_id}}'
  - Beyond the Darkness:
      identified_by: date
      set:
        path: 'C:/Media/Pool/Downloads/Downloaded/temp/{{series_name}}/{{series_name}} - {{series_id}}'

inputs:
  - rss: http://upandatemshow.com/podcast/upandatem/feed/
  - rss: http://www.podcastone.com/podcast?categoryID2=1010

download: 'C:/Media/Pool/Downloads/Downloaded/temp/{{series_name}}/{{series_name}} - {{series_id}}'
```
  
#### Log:
```
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `32: It’s All Words. It’s All Soundbites.` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `31: Good Answer! Good Answer!` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `30: I Don't Throw Parties, I am the Party` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `29: Straight Cash‚ Homie` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `28: The Glumping of 2016` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `27: Babies Eat Sanity` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `26: I Want to Keep the Terrorists Out` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `25: Murder of Goats` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `24: Hansmeier` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `23: I Didn’t Grow Up a Girl` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `22: Electors Electing Electors` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `21: Puff-Paint Paul Hornung Jersey` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `20: Start with the Butt Hurt` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `19: Air Buds` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `18: Bring Jacob Home` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `17: UberBus` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `16: Jacob` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `15: Stay Above the Doo-Doo Line` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `14: Slamfire Shotgun` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `13: The Brony File` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `12: Disco Fries` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `11: Put a Stick in There` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `10: Live on Tape from Plymouth` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `9: When You Lose the Minneapolis City Council` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `8: Podcasting is Below Ham Radio` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `7: It Was Just Like MASH` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `6: Amateur Zookeeper` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `5: Urgent Care` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `4: Radio Rejects` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `3: Routine Hydrant Flushing` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `2: Think about Hepatitis C` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `1: You’re not good enough to be here. Please leave.` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `Slender Man: From Internet Meme To Murder` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `Bigfoot: Killer Encounters` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `Sinister Intentions: Invasion Of The Shadow People` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `Black Eyed Kids: Alien Intruders?` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `Black Eyed Kids and Possession` by accept_all plugin
2017-01-20 22:26 VERBOSE  task          Podcasts        ACCEPTED: `Beyond The Darkness Trailer` by accept_all plugin
2017-01-20 22:26 INFO     download      Podcasts        Downloading: 110: Get Your Schnitte Together
2017-01-20 22:26 CRITICAL task          Podcasts        BUG: Unhandled error in plugin download: 
Traceback (most recent call last):
  File "c:\python27\lib\site-packages\flexget\task.py", line 490, in __run_plugin
    return method(*args, **kwargs)
  File "c:\python27\lib\site-packages\flexget\event.py", line 23, in __call__
    return self.func(*args, **kwargs)
  File "c:\python27\lib\site-packages\flexget\plugins\output\download.py", line 94, in on_task_download
    tmp_path=tmp)
  File "c:\python27\lib\site-packages\flexget\plugins\output\download.py", line 179, in get_temp_files
    self.get_temp_file(task, entry, require_path, handle_magnets, fail_html, tmp_path)
  File "c:\python27\lib\site-packages\flexget\plugins\output\download.py", line 132, in get_temp_file
    error = self.process_entry(task, entry, url, tmp_path)
  File "c:\python27\lib\site-packages\flexget\plugins\output\download.py", line 199, in process_entry
    self.download_entry(task, entry, url, tmp_path)
  File "c:\python27\lib\site-packages\flexget\plugins\output\download.py", line 236, in download_entry
    response = task.requests.get(url, auth=auth, raise_status=False)
  File "c:\python27\lib\site-packages\requests\sessions.py", line 501, in get
    return self.request('GET', url, **kwargs)
  File "c:\python27\lib\site-packages\flexget\utils\requests.py", line 241, in request
    log.trace('Contents for URL %s: %s', url, result.text)
  File "c:\python27\lib\site-packages\requests\models.py", line 810, in text
    encoding = self.apparent_encoding
  File "c:\python27\lib\site-packages\requests\models.py", line 680, in apparent_encoding
    return chardet.detect(self.content)['encoding']
  File "c:\python27\lib\site-packages\requests\packages\chardet\__init__.py", line 30, in detect
    u.feed(aBuf)
  File "c:\python27\lib\site-packages\requests\packages\chardet\universaldetector.py", line 128, in feed
    if prober.feed(aBuf) == constants.eFoundIt:
  File "c:\python27\lib\site-packages\requests\packages\chardet\charsetgroupprober.py", line 64, in feed
    st = prober.feed(aBuf)
  File "c:\python27\lib\site-packages\requests\packages\chardet\sjisprober.py", line 54, in feed
    for i in range(0, aLen):
MemoryError
```

### Additional information:

- Flexget Version: 2.9.5
- Python Version: Python 2.7.11
- Installation method: pip method from flexget.com install guide
- OS and version: Windows Server 2012r2, all latest patches
- Link to crash log:
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `32: It’s All Words. It’s All Soundbites.` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `31: Good Answer! Good Answer!` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `30: I Don't Throw Parties, I am the Party` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `29: Straight Cash‚ Homie` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `28: The Glumping of 2016` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `27: Babies Eat Sanity` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `26: I Want to Keep the Terrorists Out` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `25: Murder of Goats` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `24: Hansmeier` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `23: I Didn’t Grow Up a Girl` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `22: Electors Electing Electors` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `21: Puff-Paint Paul Hornung Jersey` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `20: Start with the Butt Hurt` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `19: Air Buds` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `18: Bring Jacob Home` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `17: UberBus` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `16: Jacob` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `15: Stay Above the Doo-Doo Line` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `14: Slamfire Shotgun` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `13: The Brony File` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `12: Disco Fries` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `11: Put a Stick in There` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `10: Live on Tape from Plymouth` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `9: When You Lose the Minneapolis City Council` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `8: Podcasting is Below Ham Radio` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `7: It Was Just Like MASH` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `6: Amateur Zookeeper` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `5: Urgent Care` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `4: Radio Rejects` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `3: Routine Hydrant Flushing` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `2: Think about Hepatitis C` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `1: You’re not good enough to be here. Please leave.` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `Slender Man: From Internet Meme To Murder` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `Bigfoot: Killer Encounters` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `Sinister Intentions: Invasion Of The Shadow People` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `Black Eyed Kids: Alien Intruders?` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `Black Eyed Kids and Possession` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `Beyond The Darkness Trailer` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `TWiT 597: Fark Google!` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `TWiT 596: President Zuck` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `TWiT 595: What A Year!` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `TWiT 594: What A Year!` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `TWiT 593: Coulrophobia` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `TWiT 592: McAfee for Cyber` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `TWiT 591: Bro-active` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `TWiT 590: The Lady in the Canister` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `TWiT 589: Save it for St. Patrick's Day` by accept_all plugin
2017-01-20 23:27 VERBOSE  task          Podcasts        ACCEPTED: `TWiT 588: New Phone, Who Dis?` by accept_all plugin
2017-01-20 23:27 INFO     download      Podcasts        Downloading: 110: Get Your Schnitte Together
2017-01-20 23:27 CRITICAL task          Podcasts        BUG: Unhandled error in plugin download: 
Traceback (most recent call last):
  File "c:\python27\lib\site-packages\flexget\task.py", line 490, in __run_plugin
    return method(*args, **kwargs)
  File "c:\python27\lib\site-packages\flexget\event.py", line 23, in __call__
    return self.func(*args, **kwargs)
  File "c:\python27\lib\site-packages\flexget\plugins\output\download.py", line 94, in on_task_download
    tmp_path=tmp)
  File "c:\python27\lib\site-packages\flexget\plugins\output\download.py", line 179, in get_temp_files
    self.get_temp_file(task, entry, require_path, handle_magnets, fail_html, tmp_path)
  File "c:\python27\lib\site-packages\flexget\plugins\output\download.py", line 132, in get_temp_file
    error = self.process_entry(task, entry, url, tmp_path)
  File "c:\python27\lib\site-packages\flexget\plugins\output\download.py", line 199, in process_entry
    self.download_entry(task, entry, url, tmp_path)
  File "c:\python27\lib\site-packages\flexget\plugins\output\download.py", line 236, in download_entry
    response = task.requests.get(url, auth=auth, raise_status=False)
  File "c:\python27\lib\site-packages\requests\sessions.py", line 501, in get
    return self.request('GET', url, **kwargs)
  File "c:\python27\lib\site-packages\flexget\utils\requests.py", line 241, in request
    log.trace('Contents for URL %s: %s', url, result.text)
  File "c:\python27\lib\site-packages\requests\models.py", line 810, in text
    encoding = self.apparent_encoding
  File "c:\python27\lib\site-packages\requests\models.py", line 680, in apparent_encoding
    return chardet.detect(self.content)['encoding']
  File "c:\python27\lib\site-packages\requests\packages\chardet\__init__.py", line 30, in detect
    u.feed(aBuf)
  File "c:\python27\lib\site-packages\requests\packages\chardet\universaldetector.py", line 128, in feed
    if prober.feed(aBuf) == constants.eFoundIt:
  File "c:\python27\lib\site-packages\requests\packages\chardet\charsetgroupprober.py", line 64, in feed
    st = prober.feed(aBuf)
  File "c:\python27\lib\site-packages\requests\packages\chardet\sjisprober.py", line 54, in feed
    for i in range(0, aLen):
MemoryError

http://pastebin.com/tt8gv9Z1
http://pastebin.com/qsNyi08U

**Comment by paranoidi** [comment]
Eggl the log.trace call is putting whole content of download into log. Not a good idea at all.

**Comment by gazpachoking** [comment]
Oops. Messed up the commit message. Fixed in 6ea9440bc397e8d03d318d2410f33bf32c61c152 Which reverts #1589
